avocado_vt/loader.py

```diff
@@ -123,6 +123,16 @@ def _get_parser(self):
         options_processor = VirtTestOptionsProcess(self.config)
         return options_processor.get_parser()
 
+    def _save_parser_cartesian_config(self, parser):
+        path = get_opt(self.config, 'vt.save_config')
+        if path is None:
+            return
+        with open(path, 'w') as cartesian_config:
+            cartesian_config.write("include %s\n" % parser.filename)
+            for statement in (parser.only_filters + parser.no_filters +
+                              parser.assignments):
+                cartesian_config.write("%s\n" % statement)
```

Comment:
Does this handle well conditional clauses, e.g.
```
previously_defined_variant:
    paramA = 1
```
? It seems like mostly accumulating the statements and printing them. Does it carefully follow the order of definitions?  I anticipate there could be lots of corner cases not covered by this and that it could be useful only for ones that are simple enough. What do you think?

Reply:
It doesn't because we don't produce such a variants in that code... In fact, the produced file consists only of the primary parsed cartesian config file (the one that gets a `include` in the produced file), and then the individual `only`s and `no`s that the Avocado-VT command line / config plugin code produces.

It's a bit trivial and stupid, but it's a simple enough and clear representation of what the plugin does.